examples/carbon-for-ibm-products/example-gallery/src/gallery-config/index.js

```diff
@@ -37,6 +37,8 @@ import imageTearsheet from './Tearsheet--thumbnail.png';
 import imageUserProfileImage from './UserProfileImage--thumbnail.png';
 import imageWebTerminal from './WebTerminal--thumbnail.png';
 import imagePrefixExample from './prefix-example--thumbnail.png';
+import imageReact16Example from './react-16-example--thumbnail.png';
```

Comment:
Should raise an issue to convert other examples to React 18

Reply:
https://github.com/carbon-design-system/ibm-products/issues/3363